shortapi/serializers.py

```diff
@@ -1,8 +1,22 @@
+
 from rest_framework import serializers
 from .models import Link
+from user.models import User
 
 
 class LinkSerializer(serializers.ModelSerializer):
+    # user = serializers.ReadOnlyField(source="user.email")
     class Meta:
         model=Link
-        fields = "__all__"
\ No newline at end of file
+        fields = "__all__"
+    
+
+
+class CustomUserSerializer(serializers.ModelSerializer):
+    shorten_url = serializers.ReadOnlyField(read_only=True)
```

Comment:
why does a user have a shorten url attr

Reply:
i wanted a situation where the user can see all the shorten url they have created once Authenticated.

Reply:
That may be a bad idea if you consider overfetching, bandwidth cost and pagination. besides I am not sure how this will auto fetch all shortened urls?

Reply:
yes sir i didnt consider all of this.